build.gradle.kts

```diff
@@ -528,6 +516,11 @@ tasks.register("python310PostCommit") {
   dependsOn(":sdks:python:test-suites:dataflow:py310:postCommitIT")
   dependsOn(":sdks:python:test-suites:direct:py310:postCommitIT")
   dependsOn(":sdks:python:test-suites:portable:py310:postCommitPy310")
+  // TODO: https://github.com/apache/beam/issues/22651
+  // The default container uses Python 3.10. The goal here is to
+  // duild Docker images for TensorRT tests during run time for python versions
+  // other than 3.10 and add these tests in other python postcommit suites.
+  dependsOn(":sdks:python:test-suites:dataflow:py310:inferencePostCommitIT")
```

Comment:
Looks like this change broke the TensorRT tests - https://github.com/apache/beam/actions/workflows/beam_PostCommit_Python.yml

The reason is that TensorRT depends on a docker image - https://github.com/apache/beam/blob/24a0447518463f7b1a244e02431be37f8c5d6cbc/sdks/python/test-suites/dataflow/common.gradle#L410 - which is static, not built based on the test suite its in (this is actually what the comment in this file is saying).

I think we need to use https://github.com/apache/beam/blob/24a0447518463f7b1a244e02431be37f8c5d6cbc/sdks/python/test-suites/containers/tensorrt_runinference/tensor_rt.dockerfile#L25 to build a new docker image for the tensorRT jobs and push it to `us.gcr.io/apache-beam-testing/python-postcommit-it/tensor_rt:latest`. @jrmccluskey would you mind following up and doing that? It looks like you already updated the dockerfile, so hopefully its smooth sailing